VCFv4.3.tex

```diff
@@ -503,6 +507,10 @@ \subsubsection{Genotype fields}
   All phased genotypes that do not contain a PS subfield are assumed to belong to the same phased set.
   If the genotype in the GT field is unphased, the corresponding PS field is ignored.
   The recommended convention is to use the position of the first variant in the set as the PS identifier (although this is not required).
+  \item PQL (List of integers): The list of PQs one for each phase set in PSL (encoded like PQ)
+  \item PSL (List of non-negative 32-bit Integer): The list of PSs one for each pipe ($\mid$) in the GT field, specifying the phase set for the allele prior to the pipe.
+  A given sample-genotype should not have values for both PS and PSL. 
```

Comment:
It might be worth discussing what implications the "should not have" has for rows with multiple samples, some having PSL and some PS. I'd go with a stronger "must not have".

Reply:
thanks. I'll update.